Advisors updates (#87)
percona-platform/portal-doc: docs/checks.md

Percona Monitoring and Management (PMM) offers four database Advisors to help you improve database performance: Configuration Advisors, Performance Advisors, Query Advisors and Security Advisors. 

Each Advisor includes a set of automated checks, which investigate a specific range of possible issues and areas of improvement: security threats, non-compliance issues, performance degradation, query and index optimization strategies etc.

All Advisors and their checks are hosted on Percona Platform. PMM Server automatically downloads them from here when the **Advisors** and **Telemetry** options are enabled in PMM under **Configuration > Settings > Advanced Settings**. Both options are enabled by default.

> ## Advisor check tiers and entitlements

Reviewer: ```suggestion
## Advisors plans 
```

### Built-in Advisors
By default, PMM comes with the basic checks included in the Configuration and the Security Advisors: **Configuration Versions** and **CVE Security**.
When your PMM instance is not connected to Percona Platform, PMM can only provide these built-in Anonymous Advisor checks.

> ### How to get more checks

Reviewer: ```suggestion
### How to get more advisors
```

> As soon as you connect your PMM instance to Percona Platform, PMM has access to extra checks, available for Registered PMM instances only.

Reviewer: ```suggestion
As soon as you connect your PMM instance to Percona Platform, PMM has access to extra advisors, available for Registered PMM instances only.
```

> As soon as you connect your PMM instance to Percona Platform, PMM has access to extra advisors, available for the Basic subscription plan.

Reviewer: Can we add a link to the Connection page here?

If you are a Percona customer with a Percona Customer Portal account, you get additional access to Paid checks, which offer even more advanced database health information.

Depending on the entitlements available for your Percona Account, the set of Advisor checks that PMM can download from Percona Platform differ in terms of complexity and functionality. For information about which Advisor checks are included in our subscriptions, see the [Percona Platform Subscription plans](https://www.percona.com/software/percona-platform/subscription).

> ## List of Advisor checks

Reviewer: ```suggestion
## List of Advisos
```

> Check out the complete list of checks and their availability for **Anonymous**, **Registered**, and **Paid** PMM instances in the tables below:

Reviewer: ```suggestion
Check out the complete list of checks and their availability for **Basic**, **Standard**, and **Premium** plans in the tables below:
```
